diff --git a/configure.ac b/configure.ac
index a53a77d..7a5fa75 100644
--- a/configure.ac
+++ b/configure.ac
@@ -632,7 +632,7 @@ AS_IF([test "x$with_natpmp" != xno],
 
 AC_DEFINE_UNQUOTED([HAVE_LIBNATPMP], `if test "x$with_natpmp" != xno; then echo 1; else echo 0; fi`, [Define if you have libnatpmp])
 
-AC_DEFINE_UNQUOTED([HAVE_SHM], `if test -z "${HAVE_LINUX_TRUE}" && test -z "${HAVE_ANDROID_FALSE}" ; then echo 1; else echo 0; fi`,
+AC_DEFINE_UNQUOTED([HAVE_SHM], 0,
     [Define if you have shared memory support])
 
 # DOXYGEN
