diff --git a/src/qtwrapper/configurationmanager_wrap.h b/src/qtwrapper/configurationmanager_wrap.h
index 5962cb18..f5bc9691 100644
--- a/src/qtwrapper/configurationmanager_wrap.h
+++ b/src/qtwrapper/configurationmanager_wrap.h
@@ -626,7 +626,7 @@ public Q_SLOTS: // METHODS
         return uint32_t(DRing::sendFile(dring_info, id));
     }
 
-    uint32_t dataTransferInfo(uint64_t transfer_id, DataTransferInfo& lrc_info) {
+    uint32_t dataTransferInfo(const uint64_t& transfer_id, DataTransferInfo& lrc_info) {
         DRing::DataTransferInfo dring_info;
         auto error = uint32_t(DRing::dataTransferInfo(transfer_id, dring_info));
         lrc_info.accountId = QString::fromStdString(dring_info.accountId);
@@ -641,15 +641,15 @@ public Q_SLOTS: // METHODS
         return error;
     }
 
-    uint64_t dataTransferBytesProgress(uint64_t transfer_id, int64_t& total, int64_t& progress) {
+    uint64_t dataTransferBytesProgress(const uint64_t& transfer_id, int64_t& total, int64_t& progress) {
         return uint32_t(DRing::dataTransferBytesProgress(transfer_id, total, progress));
     }
 
-    uint32_t acceptFileTransfer(uint64_t transfer_id, const QString& file_path, int64_t offset) {
+    uint32_t acceptFileTransfer(const uint64_t& transfer_id, const QString& file_path, int64_t offset) {
         return uint32_t(DRing::acceptFileTransfer(transfer_id, file_path.toStdString(), offset));
     }
 
-    uint32_t cancelDataTransfer(int64_t transfer_id) {
+    uint32_t cancelDataTransfer(const int64_t& transfer_id) {
         return uint32_t(DRing::cancelDataTransfer(transfer_id));
     }
 
